Documentation/tectonic.md

```diff
@@ -0,0 +1,52 @@
+# Rook on Tectonic
+
+Here is a running guide on how to implement Rook on Tectonic. A complete guide on how to install Tectonic is out of the scope of the Rook project. But many info can be found on the [Tectonic website](https://coreos.com/tectonic/docs/latest/)
+
+## Prequisites
+
+- An installed tectonic-installer. These steps are described on [the Tectonic website](https://coreos.com/tectonic/docs/latest/install/bare-metal/#4-tectonic-installer)
+- A running matchbox node which will do the provisioning (Bare metal)
+- You can run through all steps of the GUI installer, but in the last step, choose `Boot manually`. This way we can make the necessary changes first.
+
+## Edit the kubelet.service file to be compatible with Rook
+
+To make the kubelet compatible with Rook we need to make a few adaptions to the systemd service file generated by the Tectonic-installer.
+
+First cd into the directory where you untarred the tectonic installer and find your newly generated cluster configuration files.
+
+```
+ $ cd ~/tectonic/tectonic-installer/LINUX-OR-DARWIN/clusters
+ $ nano modules/ignition/resources/services/kubelet.service
+```
+
+And after the line `ExecStartPre=/bin/mkdir -p /var/lib/cni`, paste the following extra lines:
+
+```
+ExecStartPre=/bin/mkdir -p /var/lib/kubelet/volumeplugins
+ExecStartPre=/bin/mkdir -p /var/lib/rook
+```
+
+And a little down in the `ExecStart` section, add the following kubelet option:
+
+```
+--volume-plugin-dir=/var/lib/kubelet/volumeplugins \
+```
+
+Save the file and close.
+
+### Provision your matchbox
+
+Still from this cluster directory, invoke the following commands:
+
+```
+ $ terraform plan -var-file=terraform.tfvars platforms/metal
+ $ terraform apply -var-file=terraform.tfvars platforms/metal
+```
+
+Now if all went well the correct files were automatically provisioned to your Matchbox node and you can boot all your cluster nodes. As an experience from XenServer this iPXE boot is really slow, so you can go to grab something to eat by now. Changes are big that the `terraform apply` command will fail after one hour, but just restart the same command again to finish up.
+
+## Start Rook
+
+Follow the instructions as stated in [the installation docs](https://rook.github.io/docs/rook/master/kubernetes.html). Best thing is to provide the disks to use in the [rook-cluster.yaml](https://rook.github.io/docs/rook/master/cluster-crd.html#storage-configuration-settings)
```

Comment:
Adding something like "After the Tectonic Installer ran and the the Kubernetes cluster is now started and ready, you can now follow the rook installation guide" and a link to the `kubernetes.md` guide.